### DISCUSSION TITLE
burn and drug discovery

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#1709] [Q&A]
Hi,
I am diving into burn and candle now and want to choose one as our deep learning framework. I learn that the founder of tracel-AI worked on drug discovery before. I am very interested in the filed because I am working at bio-tech company. 
![image](https://github.com/tracel-ai/burn/assets/29703450/b164785f-2592-4031-a0d8-9ce60cd08ffc)

Do you mind sharing some experience on drug discovery (such as publications or models developed by Nathaniel simard) ?

**Comment by antimora** [comment]
This is probably a question for @louisfd

**Comment by nathanielsimard** [comment]
Hi @wangjiawen2013, the work was part of my Masters and isn't publicly available. I worked on graph transformers for chemical reaction prediction. I developed a graph-to-graph model to replace the SMILES representation, but decoding a graph was very slow and not as performant as decoding SMILES, though we kept the graph encoder since it offered better explainability (we could spot reaction centers based on attention weights, pretty nice).
